src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionServiceTest.kt

```diff
@@ -233,31 +200,48 @@ class LicenceConditionServiceTest {
     verify(auditService, times(1)).recordAuditEventDeleteAdditionalConditions(any(), any(), any())
 
     assertThat(licenceCaptor.value.additionalConditions).containsExactly(
-      AdditionalCondition(
-        id = 1,
-        conditionVersion = "1.0",
-        conditionCode = "code",
-        conditionCategory = "oldCategory",
-        conditionSequence = 5,
-        conditionText = "oldText",
-        expandedConditionText = "expandedOldText",
-        conditionType = "AP",
-        additionalConditionData = someAdditionalConditionData,
-        licence = aLicenceEntity,
+      additionalCondition(1),
+      additionalCondition(3),
+    )
+
+    // Verify last contact info is recorded
+    assertThat(licenceCaptor.value.updatedByUsername).isEqualTo("smills")
+  }
+
+  @Test
+  fun `delete multiple conditions`() {
+    whenever(communityOffenderManagerRepository.findByUsernameIgnoreCase("smills")).thenReturn(aCom)
+    val licenceEntity = aLicenceEntity.copy(
+      additionalConditions = listOf(
+        additionalCondition(1),
+        additionalCondition(2),
+        additionalCondition(3),
       ),
-      AdditionalCondition(
-        id = 3,
-        conditionVersion = "1.0",
-        conditionCode = "code3",
-        conditionCategory = "oldCategory3",
-        conditionSequence = 6,
-        conditionText = "oldText3",
-        expandedConditionText = "expandedOldText",
-        conditionType = "AP",
-        additionalConditionData = someAdditionalConditionData,
-        licence = aLicenceEntity,
+      standardConditions = listOf(
+        standardCondition(1).copy(conditionType = "AP"),
+        standardCondition(2).copy(conditionType = "AP"),
+        standardCondition(3).copy(conditionType = "PSS"),
```

Comment:
@nishanthmahasamudram I've tidied this up a bit -

previously it did not define the list of standard conditions and instead just checked that the list of standard only contained the PSS condition. By making it clearer what exists on the licence upfront it makes it clearer the behaviour you're testing